README.md

```diff
@@ -1,2 +1,5 @@
 # terra-mars-api
 Web API for returning data about Terraforming Mars games
+
+#To run (development)
```

Comment:
Nitpick: space after `#`. Also consider using [reStructuredText](https://docutils.sourceforge.io/docs/user/rst/quickref.html) since that's the standard markup lang in python :P The equivalent would probs be

```rst
==============
terra-mars-api
==============

Web API for returning data about Terraforming Mars games

To run (development)
====================

.. code:: python

    python manage.py --settings=mars.settings.development
```